### ISSUE TITLE
Q: WatchOS3 support

### ISSUE THREAD

**Issue by szuniverse** [daltoniam/Starscream#269]
Hi

As far as I know the watchOS3 support stand-alone apps, I mean you can use apps withous your iphone, am I right? if yes, do you guys planning to support watchOS 3?

**Comment by daltoniam** [comment]
Yeah, I would like to do WatchOS3 support. WatchOS3 doesn't support the CFMessage (Core Foundation) calls we use, so an investigate will have to be done to see if this can be worked around. PR is welcomed.

**Comment by hamchapman** [comment]
Both the CoreFoundation and CFNetwork docs seem to suggest (in the sidebar to the right) support fro watchOS 3 - is it some other APIs that watchOS is missing support for, or perhaps a specific subset?

https://developer.apple.com/reference/corefoundation
https://developer.apple.com/reference/cfnetwork

**Comment by daltoniam** [comment]
@hamchapman it is a subset of the APIs that are missing last I checked (been quite a few months now). It might be easy to work around the missing subset, but I haven't had the time to do enough research to see what would be need (I just added a watchOS target and saw all the build errors). Might not be a bad thing to look at watchOS and Linux at the same time, see what would need to be add/changed to get both targets working.
